### DISCUSSION TITLE
How to get QuickAdd choices into Multi?

### DISCUSSION THREAD

**Discussion by jaymaleoca** [chhoumann/quickadd#258] [Q&A] [answered]
Hi Christian and all -

First thanks for this plug in. I love it.

I can't figure out how I can put the QuickAdd choices (here in the screenshot, let's say M Note, E Note, and C Note) into the Multi option. I've tried dragging them under the newly created Multi, drag them over to the Multi, and so on. Still quite can't figure out how to get it "into" the Multi folder.

Couldn't find a discussion thread on it as well - so I don't know if that's just me being silly...

Thank you,

![image](https://user-images.githubusercontent.com/92065845/174922887-e386292e-50d5-467e-a61b-a67f26fd8016.png)

**Comment by robosquiggles** [comment]
With enough careful clicking and dragging I managed to get one item into the multi - once you get one in it's much easier to get the others in there. I think I had the ⚡ enabled on the Multi at the time - maybe that will help you out?

**Comment by robosquiggles** [comment, marked as answer]
Someone here had a useful hint for Windows https://github.com/chhoumann/quickadd/discussions/249.
Cheers!

**Reply by jaymaleoca** [reply]
Thank you, robosquiggles!

**Comment by chhoumann** [comment]
Coming back to this one very late - sorry about that.

Glad the hint from #249 got you sorted at the time. For anyone landing here now: there is a much easier way since 2.10.0 - right-click the choice in Settings -> QuickAdd and pick **Move to: [your Multi]**. No dragging needed. Dragging was also reworked since and behaves reliably now.

Docs: https://quickadd.obsidian.guide/docs/Choices/MultiChoice

Closing as resolved.
